rollup.config.js

```diff
@@ -0,0 +1,72 @@
+import babel from 'rollup-plugin-babel';
+import resolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+import filesize from 'rollup-plugin-filesize';
+import localResolve from 'rollup-plugin-local-resolve';
+import minify from 'rollup-plugin-babel-minify';
+import postcss from 'rollup-plugin-postcss';
+
+import pkg from './package.json';
+
+const externalDependencies = [
```

Comment:
[Here's](https://mixmax.com/blog/rollup-externals) a good writeup on `rollup.js` `externals` and `globals`